apis/palette_api/app/api.py

```diff
@@ -18,27 +17,21 @@
     docs_url='/docs',
     redoc_url='/redoc'
 )
-# CORS
-app.add_middleware(
-    CORSMiddleware,
-    allow_origins=['*']
-)
 
 # create API endpoints
 
 
 @app.get('/works/{catalogue_id}')
 def palette_similarity_by_catalogue_id(catalogue_id: str, n: int = 10):
-    catalogue_id = catalogue_id or np.random.choice(catalogue_ids)
-    if catalogue_id not in catalogue_ids:
+    catalogue_id = catalogue_id or np.random.choice(valid_catalogue_ids)
```

Comment:
fwiw I don't think the random choice will ever be selected as `/works/` and `/works` will 404 first, but that's fine imo as a random result for a non-specified ID is not especially RESTful